src/Storefront/Resources/app/storefront/src/plugin/address-manager/address-manager.plugin.js

```diff
@@ -0,0 +1,249 @@
+        }
+
+        this._client.get(
+            '/widgets/account/address-manager',
+            (response)  => {
+                this._renderModal(response);
+                this._registerEvents();
+            }
+        );
+    }
+
+    _renderModal(response) {
+        const pseudoModal = new PseudoModalUtil(response,);
+
+        pseudoModal.open(() => this._btnLoader.remove(), 0);
+
+        const modal = pseudoModal.getModal();
+        modal.children[0].classList.add('modal-dialog-scrollable');
+        modal.children[0].classList.add('modal-dialog-address');
+
+        if (this.options.initialTab === BILLING) {
+            document.getElementById(this.options.billingAddressTabId).checked = true;
+            this._onSwitchToBillingTab();
+        } else {
+            document.getElementById(this.options.shippingAddressTabId).checked = true;
+        }
+
+        window.PluginManager.initializePlugins();
+    }
+
+    _onSwitchToBillingTab() {
+        document.getElementById(this.options.billingAddressTabId).checked = true;
+        document.getElementById(this.options.shippingTabPaneId).classList.remove('show');
+        document.getElementById(this.options.shippingTabPaneId).classList.remove('active');
+        document.getElementById(this.options.billingTabPaneId).classList.add('show');
+        document.getElementById(this.options.billingTabPaneId).classList.add('active');
+    }
+
+    _onSwitchToShippingTab() {
+        document.getElementById(this.options.shippingAddressTabId).checked = true;
+        document.getElementById(this.options.billingTabPaneId).classList.remove('show');
+        document.getElementById(this.options.billingTabPaneId).classList.remove('active');
+        document.getElementById(this.options.shippingTabPaneId).classList.add('show');
+        document.getElementById(this.options.shippingTabPaneId).classList.add('active');
+    }
+
+    _onSaveChanges(event) {
+        event.preventDefault();
+
+        try {
+            this._btnLoader = new ButtonLoadingIndicatorUtil(this.options.addressEditSubmitSelector);
+            this._btnLoader.create();
+        } catch (error) {
+            console.warn('[AddressManagerPlugin] Unable to create loading indicator on button', error);
+        }
+
+        this._client.post(
+            event.currentTarget.action,
+            new FormData(event.target),
+            (data, response)  => {
+                if (response.status === 204) {
+                    location.reload();
+                } else {
+                    this._replaceModalContent(data);
+                }
+            }
+        );
+    }
+
+    _onSetRadioInputActive(element, event){
+        if (event.target.closest('.dropdown')) {
+            return;
+        }
+
+        if (!(element instanceof HTMLElement)) {
+            console.warn('[AddressManagerPlugin] Invalid HTML element provided.');
```

Comment:
What? where? how? if you're logging something in stringified content, you need to know what it's about.

Besides that, your code flow willl never end in this situation, as it gets created from `querySelectorAll`, therefore remove it and use a doc type instead:

```js
/**
  * @param {HTMLElement} element
 */
_onSetRadioInputActive(element, event){
```